apps/docs/content/components/CTAForm.mdx

```diff
@@ -0,0 +1,121 @@
+      <Checkbox name="confirm" />
+    </FormControl>
+  </CTAForm.Confirm>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+## Component props
+
+### CTAForm
+
+| name        | type     | default     | required | description                                                        |
+| ----------- | -------- | ----------- | -------- | ------------------------------------------------------------------ |
+| `action`    | `string` | `undefined` | `false`  | The URL that processes the form submission                         |
+| `method`    | `string` | `undefined` | `false`  | The HTTP method to submit the form with                            |
+| `target`    | `string` | `undefined` | `false`  | Indicates where to display the response after submitting the form. |
+| `className` | `string` | `undefined` | `false`  | Sets a custom class                                                |
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your Work Email Address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+### CTAForm.Input
+
+| name        | type     | default     | required | description         |
+| ----------- | -------- | ----------- | -------- | ------------------- |
+| `className` | `string` | `undefined` | `false`  | Sets a custom class |
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your Work Email Address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Confirm>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>
+        I agree to the <InlineLink href="#">Privacy Policy</InlineLink> and{' '}
+        <InlineLink href="#">Terms of Service</InlineLink>
+      </FormControl.Label>
+      <Checkbox name="confirm" />
+    </FormControl>
+  </CTAForm.Confirm>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+### CTAForm.Confirm
+
+| name        | type     | default     | required | description         |
+| ----------- | -------- | ----------- | -------- | ------------------- |
+| `className` | `string` | `undefined` | `false`  | Sets a custom class |
+
+```jsx live
+<CTAForm>
+  <CTAForm.Input>
+    <FormControl required size="large" fullWidth>
+      <FormControl.Label>Your Work Email Address</FormControl.Label>
+      <TextInput placeholder="name" />
+    </FormControl>
+  </CTAForm.Input>
+  <CTAForm.Action>Subscribe</CTAForm.Action>
+</CTAForm>
+```
+
+### CTAForm.Action
+
+| name        | type     | default     | required | description         |
+| ----------- | -------- | ----------- | -------- | ------------------- |
+| `className` | `string` | `undefined` | `false`  | Sets a custom class |
+
+{/* TODO: Add all button properties here */}
```

Comment:
Just a reminder to remove this, it's currently in the docs. Should this be a HTML comment instead?
![Screenshot 2023-02-13 at 13 42 29](https://user-images.githubusercontent.com/13340707/218474113-396782da-45bd-43e3-b661-bb4553cb94d3.png)

Reply:
This is a good note. I wanted to document somewhere that when the Button docs get filled out this should be updated. Maybe I could include it on the issue for that work. I believe Tyler will be taking that on correct?

Reply:
Adding to issue sounds good. We shouldn't render these as markup in the user-facing docs. If you can turn it into a comment that'd be great 👍